packages/labs/src/grpc_web/change_import_style.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * Converts a list of generated protobuf-js files from commonjs modules into named AMD modules.
```

Comment:
I think you got this code or approach from somewhere? could you include a link so we can find that reference later?
(or add a README for us to understand the design)

Reply:
Done! Added a nice little readme